### ISSUE TITLE
Error in from_file function

### ISSUE THREAD

**Issue by canuse** [Ledenel/auto-white-reimu#17]
Function `from_file(file)` in `mahjong.record.reader.py` will cause a TypeError at `mahjong.record.reader.py, line 117` when passing the filename into the function.

**Comment by canuse** [comment]
An example:
```
Traceback (most recent call last):
  File "C:\Users\canuse\AppData\Local\JetBrains\Toolbox\apps\PyCharm-P\ch-0\192.7142.42\helpers\pydev\pydevd.py", line 1415, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "C:\Users\canuse\AppData\Local\JetBrains\Toolbox\apps\PyCharm-P\ch-0\192.7142.42\helpers\pydev\_pydev_imps\_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "C:/Users/canuse/PycharmProjects/auto-white-reimu/mahjong/tenhou_record_check.py", line 246, in <module>
    main()
  File "C:/Users/canuse/PycharmProjects/auto-white-reimu/mahjong/tenhou_record_check.py", line 118, in main
    record = from_file("2019010100gm-00a9-0000-1f1a5f1f.xml")
  File "C:\Users\canuse\PycharmProjects\auto-white-reimu\mahjong\record\reader.py", line 164, in from_file
    return TenhouRecord(ET.parse(file))
  File "C:\Users\canuse\PycharmProjects\auto-white-reimu\mahjong\record\reader.py", line 117, in __init__
    events = [TenhouEvent(event) for event in events]
TypeError: 'ElementTree' object is not iterable
```

**Comment by Ledenel** [comment]
Sorry about the method `from_file` not even be coveraged in test, and actually I've never used it.

I've not recognized that the `ET.parse` returns differnet things from `ET.fromstring`. 
I checked the `ElementTree` object returned by `ET.parse`, 
assume it is `et`, and we can get root element same as `ET.fromstring` by:

1. get it's iterator by `et.iter()`
2. pick the first value of it, means `next(et.iter())`
3. pass it to the `TenhouRecord()` constructor, and hopefully every thing will be fine.

**Comment by Ledenel** [comment]
fixed in #18 .
Thanks for the contribution!
